docs/development_guide.md

```diff
@@ -43,24 +43,39 @@ In summary, Decidim developers that work on `feature/...` or `fix/...` branches
 
 Then, to start a new feature branch off from `develop` in the following way:
 
-```
+```bash
 git checkout develop
 git checkout -b feature/xxx
 ```
 
 Implement the feature, and open a Pull Request as normal, but against `develop` branch. As this is the most common operation, `develop` is the default branch instead of `master`.
 
+### Naming Decidim branches
+
+We would like to have all branches following this namings:
+
+| Branch prefix | Comment |
+| -------- | -------- |
+| chore/     | Internal work. For instance, CI, automatisms, etc. No production code change.     |
+| doc/ | For changes to the documentation. |
+| feature/ | For new features for the users or gem Decidim itself.  |
```

Comment:
It looks ok for me @tramuntanal 
Only a minor thing appart from @andreslucena [adding branch prefixs suggestion](https://github.com/decidim/decidim/pull/6210#discussion_r442010112) :
*  If we use short prefix ```fix``` instead of ```bugfix```, here maybe it would be better to use ```feat``` short prefix instead of ```feature``` too? Or in other words, I think it could be better to use both short prefixes or both long prefixes, but not mixed ones...

Reply:
I've used "fix" because it is shorter but not as a short suffix for "bugfix". Branch names can grow long, this is why I preferred the sorter word..